definitions/apm-application/relationships.yml

```diff
@@ -2,7 +2,7 @@ relationships:
   # Select entityGuid, peer.hostname from Span where peer.hostname is not null and category =
```

Comment:
I would personally move the relationship synthesis definitions into `definitions.yml`.

Reply:
I've no strong opinion. It was originally there and suggested to move it out of there so i complied. Also, it follows convention as is.